lib/CL/clSetMemObjectDestructorCallback.c

```diff
@@ -5,7 +5,20 @@ POname(clSetMemObjectDestructorCallback)(  cl_mem  memobj ,
                                     void (CL_CALLBACK * pfn_notify)( cl_mem /* memobj */, void* /*user_data*/), 
                                     void * user_data  )             CL_API_SUFFIX__VERSION_1_1
 {
-  POCL_ABORT_UNIMPLEMENTED("The entire clSetMemObjectDestructorCallback call");
+  mem_destructor_callback_t *callback;
+
+  POCL_RETURN_ERROR_COND((memobj == NULL), CL_INVALID_MEM_OBJECT);
+  POCL_RETURN_ERROR_COND((pfn_notify == NULL), CL_INVALID_VALUE);
+
+  callback = malloc(sizeof(mem_destructor_callback_t));
```

Comment:
This should be malloc (sizeof (mem....)   in the GNU C coding style. Similar to other argument lists.